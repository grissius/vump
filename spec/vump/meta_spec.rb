require 'vump/meta'

RSpec.describe 'Vump meta data' do
  it 'works and is unchanged' do
    meta = %i[version author name description email license homepage summary]
           .map { |m| [m, Vump::Meta.public_send(m)] }.to_h
    expect(meta).to match_snapshot
  end
end
