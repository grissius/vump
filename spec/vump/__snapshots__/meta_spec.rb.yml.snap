---
"./spec/vump/meta_spec.rb[1:1]":
  :author: Jaroslav Šmolík
  :name: vump
  :description: Semantic version bumper. CLI tool to easily manage projects using
    semver and automate menial version raising.
  :email: grissius@gmail.com
  :license: MIT
  :homepage: https://github.com/grissius/vump
  :summary: Raise version version in root of semver project.
